### DISCUSSION TITLE
Low latency video streaming over local network using MediaMTX

### DISCUSSION THREAD

**Discussion by nGndock** [bluenviron/mediamtx#3542] [Questions]
### Question

I'm a system admin in a small company (my first job). I'm using OBS with MediaMTX to stream content to less than 5 clients within local network. All devices are capable of H265 decoding. However I am having issues with latency. RTMP/RTSP/HLS or WebRTC no matter which protocol I use, there is a 10 second delay. What can I do to decrease the latency? Need low latency (2 - 3 secs is ok.)

**Comment by asturl** [comment]
This seems like an OBS/source issue because nothing I ever passed through MediaMTX (RTSP/RTMP in, RTSP/WebRTC out) has received any notable delay (realtime playback in the low hundred milliseconds) with standard settings.

Any more details about your pipeline/setup?

**Comment by aler9** [comment]
This is impossible to debug without details on the setup. If you are still having troubles, open another discussion and provide details on the issue, what software are you using to publish and read the stream and server logs.
